spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceExecution.scala

```diff
@@ -22,7 +22,7 @@ import org.apache.hadoop.fs.FileSystem
 import org.apache.spark.sql.functions.{lit, to_date}
 import org.apache.spark.sql.{DataFrame, SparkSession}
 import za.co.absa.atum.AtumImplicits
```

Comment:
Not needed. Just need to remove `AtumImplicits.SparkSessionWrapper`  on line 117 and 122

Reply:
Thanks for noticing, done.